### PR TITLE
Je styling 2

### PR DESCRIPTION
**Pull request by skratz17**
This is another styling branch. Basically just check it out and make sure everything looks good / stuff is positioned better than last time / stuff is reasonable and at least mostly responsive when you resize the browser window / etc. I did touch a lot of files in this branch, but all the changes were either:

- Wrapping components within Bootstrap layout components (like Row / Col) or changing existing divs to such layout components
- Adding bootstrap css classNames to components
- Removing custom CSS where no longer necessary due to bootstrap classes / new styling choices
- Like maybe two instances of inlined styles in Components (e.g., via the `styles` prop)

### CODE REVIEW

**Comment by gavinschriver** [comment]
THIS ALL LOOKS INCREDIBLE ALSO CAN WE TALK ABOUT HOW POST TAG FORM IS MAPPING AN ARRAY OF OBJECTS TO CREATE AN ARRAY OF FUCKING PROMISES TWICE

oh also.... if you confirm delete on everyones fave component but in a non-operational situation (category and tag lists) you get an error caaussee hes trying to call a delete function prop that doesnt exist so maaybeee call like a fake function there? maybe hwo knows whatever

File: src/components/comments/CommentList.js [hunk 1]
```diff
@@ -13,9 +13,8 @@ export const CommentList = props => {
   }, [])
 
   return (
-    <div>
-      <h3 className="text-center my-3">Comments</h3>
-      <ListGroup className="w-75 mx-auto">
+    <div className="commentList" style={{ maxHeight: '250px', overflowY: 'scroll' }}>
```
[the remark this answers is not in the document]
Comment (mgjeffries) [reply src/components/comments/CommentList.js:16]: I'm ok with leaving the comment scrolling for the next pr

**Review by mgjeffries — APPROVED** [review]
Looks great!

**Review by gavinschriver — APPROVED** [review]

**Review by mgjeffries — APPROVED** [review]
Tested word break in comment content

**Pull request merged**